### ISSUE TITLE
Use floats for attributes labels

### ISSUE THREAD

**Issue by vitor-vfm** [ECE493Group3/personal-stylist-NN#2]
Change [-1, 0, 1] to [0.0, 0.5, 1.0] in the neural net. It will make more sense to calculate the loss function this way.

**Comment by vitor-vfm** [comment]
Fixed with 2f213bfac047d3bca11837329ee95a1be308ae50
